src/style.css

```
@@ -1,3 +1,76 @@
+/* #region Reset */
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+a {
+  cursor: pointer;
+}
+/* #endregion */
+
+.navigation {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: auto;
+  padding: 0 50px;
+}
+
+.nav-logo {
+  display: block;
+  margin-top: 5px;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  text-transform: uppercase;
+  white-space: nowrap;
+  margin: 0;
+  padding: 0;
+}
+
+.link {
+  color: black;
+  text-decoration: none;
+  display: block;
+  position: relative;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+}
+
+.center {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.left {
+  margin-right: 10px;
+}
+
+.right {
+  margin-left: 10px;
+}
```

please use pseudo classes here e.g
```suggestion
.link:not(:last-of-type){
  margin-left: 10px;
}
```

---

Hello, I have tryed to replace it with :firstchilde :lastchild, but otherwise it's not working... :C